core/src/main/java/edu/wpi/grip/core/operations/opencv/NewSizeOperation.java

```diff
@@ -41,6 +41,8 @@ public NewSizeOperation(InputSocket.Factory inputSocketFactory,
     this.heightSocket = inputSocketFactory.create(heightHint);
 
     this.outputSocket = outputSocketFactory.create(outputHint);
+    
+    this.perform();//So initial value is correct.
```

Comment:
Working on this now. Is it better to add a parameter to the existing create size socket hint or just create a new method?